### PR TITLE
Added example for Ember Octane

### PR DESCRIPTION
**Pull request by DavertMik**
Added a small example for Ember Octane. I hope that saves time for others

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,17 @@ Standard example:
 {{/x-tree}}
 ```
 
+Ember Octane example:
+
+```handlebars
+<XTree @model={{ this.content }} as |node|>
```
Comment (btecu) [inline README.md]: The spaces around is not needed

File: README.md [hunk 2]
```diff
@@ -44,6 +44,17 @@ Standard example:
...
+  {{node.toggle}}
+  {{node.checkbox}}
+  {{node.model.name}}
+
```
Comment (btecu) [inline README.md]: Remove

**Comment by DavertMik** [comment]
Done!

**Comment by btecu** [comment]
Thanks!

**Pull request merged**